lasagne/tests/layers/test_pool.py

```diff
@@ -469,6 +482,75 @@ def test_get_output_shape_for(self, input_shape, output_shape):
         #    pytest.skip()
 
 
+class TestUpscale2DLayer:
+    def scale_factor_test_sets():
+        for scale_factor in [2, 3]:
+                yield scale_factor
+
+    def input_layer(self, output_shape):
+        return Mock(get_output_shape=lambda: output_shape,
+                    output_shape=output_shape)
+
+    def layer(self, input_layer, scale_factor):
+        from lasagne.layers.pool import Upscale2DLayer
+        return Upscale2DLayer(
+            input_layer,
+            scale_factor=scale_factor,
+        )
+
+    def test_invalid_scale_factor(self):
+        from lasagne.layers.pool import Upscale2DLayer
+        from mock import Mock
```

Comment:
This import is unused, please remove! (Also it's redundant, `Mock` is imported at the top of the file.)